dev/AppNotifications/NotificationDevicesData.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation and Contributors.
+// Licensed under the MIT License.
+
+#include "pch.h"
+
+#include "AppNotificationDevicesData.h"
+#include "NotificationDevicesData.h"
+#include "AppNotificationUtility.h"
+
+using namespace Microsoft::Windows::AppNotifications::Helpers;
+
+NotificationDevicesData::NotificationDevicesData(winrt::AppNotificationDevicesData const& devicesData)
+    : m_devicesData(devicesData)
+{
+}
+
+STDMETHODIMP NotificationDevicesData::get_CameraDeviceId(_Out_ HSTRING* value) noexcept try
```

Comment:
Out of curiosity, why WRL and "ABI WinRT" instead of C++/WinRT here?

Reply:
The "NotificationDevicesData " is implementing the "IToastDeviceData" interface that's defined in framework udk package (\Packages\Microsoft.FrameworkUdk\1.7.0-CI-26107.1717.240923-1029.1\include\FrameworkUdk\ToastNotificationsRT.h). This is done in similar to other interface definitions from frameworkudk to maintain consistency with existing UDK interfaces.